packages/insomnia/src/ui/components/rendered-query-string.tsx

```diff
@@ -33,24 +33,38 @@ interface Props {
 
 const defaultPreview = '...';
 
+function getQueryParamsFromAuth(auth: RequestAuthentication | {}): RequestParameter[] {
+  if (!('type' in auth)) {
```

Comment:
`if (!auth.type) {}` seems easier to read

Reply:
I built a helper for this `getAuthObjectOrNull` if this equals null that means auth is not set or empty object or doesn't have .type I've made a suggestion for the change but please test it to confirm. These cases are all possible because request groups can have auth objects but existing ones are uninitialised and therefore undefined.
```suggestion
  if (getAuthObjectOrNull(auth) === null) {
```

Reply:
Sure, let me also test this later.

Reply:
this also should preview auth fields coming from folders above